artificial-scrum-master/apps/artificial.scrum.master.frontend/src/app/features/project/components/sprint-preview-item/sprint-preview-item.component.html

```diff
@@ -1,16 +1,23 @@
 <div class="flex flex-col divide-y mx-2">
   <div
-    class="flex flex-col p-4 outline outline-1 outline-gray-200 shadow-lg my-1 hover:bg-zinc-100 hover:cursor-pointer"
+    class="flex flex-col p-4 outline outline-1 outline-gray-200 shadow-lg my-1 hover:cursor-pointer"
     (click)="
       redirectToUserStories(sprintElement.projectId, sprintElement.sprintId)
     "
   >
-    <span class="mat-headline-8 dark:text-gray-700"
-      >{{ sprintElement.projectName }}
-    </span>
-    <span class="mat-headline-6 font-bold dark:text-gray-700"
-      >{{ sprintElement.sprintName }}
-    </span>
+    <div class="flex flex-row justify-between">
+      <div class="flex flex-col">
+        <span class="mat-headline-8">{{ sprintElement.projectName }} </span>
+        <span class="mat-headline-6 font-bold"
+          >{{ sprintElement.sprintName }}
+        </span>
+      </div>
+      <div class="items-end">
+        <a mat-icon-button (click)="openLink($event)" target="_blank">
```

Comment:
Możesz zrobić takie coś:
```suggestion
        <a mat-icon-button [href]="url" target="_blank">
```

a w typescripcie
```
public get url() {
  return "twoj url";
}
```